server/database/queries/house/getNewestHousesData.js

```diff
@@ -3,7 +3,7 @@ const connection = require('../../config/connection');
 const getNewestHousesdata = (limit = 10, skip = 0) => {
   const sql = {
     text:
-      'SELECT * FROM (SELECT * FROM houses ORDER BY id DESC LIMIT $1 OFFSET $2) newest ORDER BY id ASC;',
+      'SELECT nh.*, u.username, u.email, u.mobile FROM (SELECT * FROM houses ORDER BY id DESC LIMIT $1 OFFSET $2) AS nh INNER JOIN users AS u ON nh.user_id = u.id ORDER BY id ASC;',
```

Comment:
this query doesn't work as expected (make a test)

Reply:
It works fine.